alibi_detect/cd/sklearn/classifier.py

```diff
@@ -134,6 +139,30 @@ def _clone_model(self):
                 model.warm_start = False
                 logger.warning('`retrain_from_scratch=True` sets automatically the parameter `warm_start=False`.')
 
+        # oob checks
+        if self.use_oob:
+            if not isinstance(model, RandomForestClassifier):
+                raise ValueError('OOB supported only for RandomForestClassifier. '
+                                 f'Received a model of type {model.__class__.__name__}')
+
+            if self.use_calibration:
+                self.use_calibration = False
+                logger.warning(
+                    'Calibration cannot be user when `use_oob=True`. Setting `use_calibration=False`.'
+                )
+
+            model.oob_score = True
+            model.bootstrap = True
+            logger.warning(
+                '`use_oob=True` sets automatically the parameter `boostrap=True` and `oob_score=True`. '
+                '`train_size` and `n_folds` are ignored when `use_oob=True`.'
```

Comment:
Would be a tiny bit more explicit, i.e. "parameters" --> "classifier parameters" as it wasn't clear without reading the code that these are not our parameters.